index.html

```diff
@@ -5,8 +5,183 @@
+                    </section>
+                </article>
+                <article class="box-9-article-2">
+                    <section class="box-9-box-content">
+                        <div class="box-9-img"></div>
+                        <h1 class="box-9-content-title">
+                            Ut in posuere nisl.<br>Aenean a urna sit amet.
+                        </h1>
+                    </section>
+                </article>
+                <article class="box-9-article-3">
+                    <section class="box-9-box-content">
+                        <div class="box-9-img"></div>
+                        <h1 class="box-9-content-title">
+                            Ut in posuere nisl.<br>Aenean a urna sit amet.
+                        </h1>
+                    </section>
+                </article>
+                <article class="box-9-article-4">
+                    <section class="box-9-box-content">
+                        <div class="box-9-img"></div>
+                        <h1 class="box-9-content-title">
+                            Ut in posuere nisl.<br>Aenean a urna sit amet.
+                        </h1>
+                    </section>
+                </article>
+                <article class="box-9-article-5">
+                    <section class="box-9-box-content">
+                        <div class="box-9-img"></div>
+                        <h1 class="box-9-content-title">
+                            Ut in posuere nisl.<br>Aenean a urna sit amet.
+                        </h1>
+                    </section>
+                </article>
+                <article class="box-9-article-6">
+                    <section class="box-9-box-content">
+                        <div class="box-9-img"></div>
+                        <h1 class="box-9-content-title">
+                            Ut in posuere nisl.<br>Aenean a urna sit amet.
+                        </h1>
+                    </section>
+                </article>
+            </section>
+            <section class="box-10">
+                <nav class="box-10-nav">
+                    <ul>
+                        <li><a href="#">Spicy </a></li>
+                        <li><a href="#">Salami</a></li>
+                        <li><a href="#">Spicy</a></li>
+                        <li><a href="#">Salami</a></li>
+                        <li><a href="#">Spicy</a></li>
+                        <li><a href="#">Salami</a></li>
+                        <li><a href="#">Spicy</a></li>
+                        <li><a href="#">Salami</a></li>
+                    </ul>
+                </nav>
+                <footer class="footer">
+                    <p>Spicy jalapeno bacon ipsum dolor amet <br> swine ham ground round kevin tail salami <br> tenderloin cow burgdoggen. Short loin andouille fatback <br> venison pork belly. Filet mignon brisket alcatra pork belly,<br> boudin spare ribs pork loin fatback chicken pork <br>chop kevin prosciutto.</p>
```

Comment:
As was said in the first review, you should use `<ul>` here and list all links with `<li>` items. Don't forget that they must be centralized inside the footer, both horizontally and vertically.

Reply:
footer has been updated with links